deploy/ocm-agent-operator-managednotifications/10-managednotifications-cr.yaml

```diff
@@ -11,6 +11,12 @@ spec:
       resendWait: 24
       severity: Info
       summary: "Multiple ingress controllers detected"
+    - activeBody: |-
+        Your cluster is currently alerting with 'KubePersistentVolumeFillingUp' in '${namespace}' for PVC '${persistentvolumeclaim}'. This is not an actionable alert for the SRE team. Please address this capacity issue using this reference documentation: https://docs.openshift.com/container-platform/latest/storage/expanding-persistent-volumes.html.
+      name: KubePersistentVolumeFillingUp
+      resendWait: 1
+      severity: Error
+      summary: Workload preventing machine deletion
```

Comment:
is this summary intended here? I would expect it to say something like 'Action required: Persistent Volume filling up' or is this somehow bound to machine deletion? :thinking:

Reply:
Not sure to understand you... but essentially the above SL comes from here:
https://github.com/openshift/managed-notifications/blob/9b9e1f9f7dcc2df2dd583427fcb2b6d924018c1b/osd/KubePersistentVolumeFillingUpError-user.json#L5

The only purpose of this PR is to automate some message currently manually sent by some SRE.

Reply:
Sorry :D 
I mean the summary field here is currently "summary: Workload preventing machine deletion",
while in the SL you linked its "summary": "Action required: Persistent Volume filling"

Reply:
Oups, you are right, thanks for noticing it :)

Reply:
@RaphaelBut : PR updated.